app/workers/notification_workers/emotion_selection_notification_worker.rb

```diff
@@ -0,0 +1,17 @@
+module NotificationWorkers
+  class EmotionSelectionNotificationWorker
+
+    attr_reader :users, :time_period
+
+    def initialize
+      @users = User.opt_in
+      @time_period = TimePeriod.last
+    end
+
+    def run_notification
+      @users.each do |user|
```

Comment:
What about making sure that the emails don't go out unless the day matches the ENV variable? Maybe between lines 11-12 add a `return unless send_emails_today?` with a private method that fetches the ENV variable and determines if today is the correct day to send them out.

Reply:
@rogergraves I think it's more appropriate to do something similar in another task - https://cloverpop-internship.atlassian.net/browse/CI-30